.terraform-version

```diff
@@ -0,0 +1 @@
+0.12.26
```

Comment:
```suggestion
0.12.19
```

match `version.tf`

Reply:
The .terraform-version is only for tfenv, and we generally try to run the latest, not the minimum compatible. Minimum compatibility is there in part to set a baseline for ppl consuming the role.